CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Maintainers
+
+## The following maintainers, listed in alphabetical order, own everything
+
+* @AliceProxy @arkodg @skriss @Xunzhuo @youngnick @zirain
```

Comment:
Github is showing an error here; Should this be `@envoyproxy/gateway-maintainers` instead?

Reply:
Yep, this is an alternative approach too.

Reply:
I think that at a minimum, we should resolve the error in the Github interface - It's saying that @AliceProxy doesn't have write access to the repo.

Reply:
Done. Thanks